src/components/ReactGoogleMaps/ReactGoogleMaps.js

```diff
@@ -342,10 +344,15 @@ export class ReactGoogleMaps extends Component {
           </div>
         </ReactTouchEvents>
         <div className={styles.searchBarContainer}>
```

Comment:
I like the idea of using the `Stack` component to group elements horizontally! Could we try to apply it to vertically stack things on the bottom-left of the Desktop view? I'm thinking this may be a way to simplify things from a CSS perspective.

Reply:
I added a Stack to group the items in the bottom-left of the Desktop view. Let me know how it looks